buildlib/pr/io_demo/az-stage-io-demo.yaml

```diff
@@ -123,7 +123,6 @@ steps:
           echo "local_ack_timeout_err counter was increased during iodemo with interference"
         else
           azure_log_error "local_ack_timeout_err counter was not increased ($(local_ack_t_err_counter_start))"
-          exit 1
```

Comment:
shall we use azure_complete_with_issues?

Reply:
Done, 10x